appveyor.yml

```diff
@@ -50,7 +50,7 @@ install:
 # - ps: .\dotnet-install.ps1 --Version 3.1.402
 
 build_script:
-  - dotnet build -c Release
+  - dotnet build -c Release /p:SourceRevisionId=$env:APPVEYOR_REPO_COMMIT /p:GitBranch=$env:APPVEYOR_REPO_BRANCH
```

Comment:
Isn't the plugin able to find these by itself?

Reply:
Same as above. current commit and current branch is available as variable in CI